### DISCUSSION TITLE
DECIMALV3 有效数字超过设置的大小，能正常落库

### DISCUSSION THREAD

**Discussion by SamuelBin** [apache/doris#18193] [A - General / Q&A] [answered]
![image](https://user-images.githubusercontent.com/34790156/228414882-b4805228-89e8-44dc-9d26-56c4b7d2fb3c.png)
设置的DECIMALV3(24,2)

但是Doris表里能写入。 9999999999999998758486016.00

![image](https://user-images.githubusercontent.com/34790156/228415423-5679d5fb-a2f7-4c4e-8968-9c3be84a7bec.png)

**Comment by LemonLiTree** [comment, marked as answer]
fix
https://github.com/apache/doris/pull/18336
